joomla-extensions/patchtester
# Properly escape \"

Pull Request for Issue https://github.com/joomla-extensions/patchtester/issues/330.

#### Summary of Changes
Properly escaping " with \"

#### Testing Instructions
Install Patchtester and inspect the "Go to Joomla! Patch Tester" link. The alt should be fully visible and the link working.
Obviously, translations need to be updated as well (original issue is for the german translation where still _QQ_ is used).
